crates/pink/pink-extension/src/lib.rs

```diff
@@ -40,6 +41,17 @@ pub enum PinkEvent {
     OspMessage(OspMessage),
     /// Contract has an on_block_end ink message and will emit this event on instantiation.
     OnBlockEndSelector(u32),
+    /// Start to transfer sidevm code
+    StartToTransferSidevmCode,
+    /// One chunk of sidevm code
+    SidevmCodeChunk(Cow<'static, [u8]>),
+    /// Start the side VM.
+    StartSidevm {
+        /// Number of memory wasm pages (64KB per page) to allocate for the side VM.
```

Comment:
It will do nothing at the moment.
If the required memory_pages out of quota, it would fail to start and we should leave it stopped.
If the OOM happens in a middle of running, the guest would panic, I am considering to restart the instance as the above comment said.